kedro-telemetry/README.md

```diff
@@ -5,23 +5,25 @@
 
-To find out how Kedro's features are used, the [Kedro team](https://github.com/kedro-org/kedro) have created the Kedro-Telemetry [plugin](https://kedro.readthedocs.io/en/stable/07_extend_kedro/04_plugins.html) to gather anonymised usage analytics. The data it gathers will help drive future development of Kedro because we can understand how the product is being used. Kedro-Telemetry uses [`pluggy`](https://pypi.org/project/pluggy/) hooks and [`requests`](https://pypi.org/project/requests/) to send data to [Heap Analytics](https://heap.io/).
+## Introduction
 
-## Privacy notice
+To find out how Kedro's features are used, the [Kedro team](https://github.com/kedro-org/kedro) have created an opt-in the Kedro-Telemetry [plugin](https://kedro.readthedocs.io/en/stable/07_extend_kedro/04_plugins.html) to gather anonymised (before being sent across the internet) and aggregated usage analytics.
 
-### What about my personal data?
+The data it gathers will help drive future development of Kedro because we can understand how the product is being used.
 
-**McKinsey & Company Inc** (“McKinsey”, “we”) will process a minimal amount of personal data when you use **Kedro** with the Kedro-Telemetry plugin. McKinsey will only process such data under applicable data protection laws.
+## Why is my data being collected?
 
-McKinsey will process your hashed hostname and hashed computer username, which both is pseudonymized information that indirectly relates to you personally. We use this data to better understand how users use **Kedro’s** features, involving a third-party analytics provider. The aim is to enable constant improvement to make **Kedro** as user friendly, effective, and precise as possible for you and other users. Furthermore, we compare your hashed hostname and hashed computer username with our internal list to understand the proportion of McKinsey users vs. open-source users. For EU users, we have implemented additional data protection safeguards, where these processing activities include a data transfer to a third country. **As an open-source user, please note that McKinsey or the third-party analytics provider will not, at any time, have the possibility to identify you personally behind the hashed hostname or hashed username (f. ex. your name, location, etc.). **As a McKinsey user, you, and the engagement you are using **Kedro** for will be identified in internal systems to evaluate McKinsey’s internal use of **Kedro**.
+Telemetry data is made available to project maintainers of the Kedro Project. The purpose of providing this information is to provide metrics to the maintainers on how Kedro is used. This data helps inform future product development because we can better understand if certain features are having traction with our users. We want to make sure that Kedro is as streamlined as possible and it is difficult to do this without usage analytics.
```

Comment:
```suggestion
Telemetry data is made available to [project maintainers of the Kedro Project](https://kedro.readthedocs.io/en/stable/faq/faq.html#who-maintains-kedro). The purpose of providing this information is to provide metrics to the maintainers on how Kedro is used. This data helps inform future product development because we can better understand if certain features are having traction with our users. We want to make sure that Kedro is as streamlined as possible and it is difficult to do this without usage analytics.
```